libs/utils/decorate.js

```diff
@@ -309,3 +307,22 @@ export function decorateMultiViewport(el) {
   }
   return foreground;
 }
+
+export function decorateAnchorVideo({ src, anchorTag }) {
+  if (!src.length || !(anchorTag instanceof HTMLElement)) return;
```

Comment:
Is there a check for this fork in the tests where the video will not get decorated? I did not see one, but it could have already been present.

Reply:
Added